### PR TITLE
Add hello world application for flex deploy check

### PR DESCRIPTION
**Pull request by nkubala**
This is taken from https://github.com/GoogleCloudPlatform/getting-started-php, and is used to ensure basic flex deploys work as intended.

@tmatsuo

### CODE REVIEW

File: testapps/deploy_check/test/app-e2e.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+runtime: php
+vm: true
```
Comment (tmatsuo) [inline testapps/deploy_check/test/app-e2e.yaml]: Do you need this file?

**Comment by nkubala** [comment]
> So my question is, do we really need another e2e test?

Possibly no. I would say PHP is unique in already having a set of apps that are deployed and tracked, so props to you guys :) That being said, I guess the name of this check is a bit misleading: we also want to check that the application serves, not just that it deploys correctly. In some sense this is similar to the integration tests, but this is using the default production builder flow (whereas the integration tests are using a custom flow on a newly built runtime image). It doesn't look like any of the tests you have currently set up are actually checking to see that the deployed application serves. WDYT?

File: testapps/deploy_check/test/app-e2e.yaml (hunk 1 above)
Comment (tmatsuo) [inline testapps/deploy_check/test/app-e2e.yaml]: Probably no, let's delete this

File: testapps/deploy_check/test/functional/ControllersTest.php [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<?php
```
Comment (tmatsuo) [inline testapps/deploy_check/test/functional/ControllersTest.php]: I don't think we need this test.

**Comment by tmatsuo** [comment]
@nkubala I think our acceptance tests are using the build_pipeline and build_pipeline56 test apps against the production builder flow, also the test checks the app serving too.

Adding @chingor for confirm

**Comment by chingor13** [comment]
@nkubala Our e2e tests do both deploys and can some simple web serving checks which can easily be added to. Our e2e tests are used for acceptance by utilizing the released builder pipelines, but are also used as CI tests for changes to our builder pipeline.

I'm not sure if this test app is needed.

**Comment by nkubala** [comment]
Fair enough, thanks for the responses

**Pull request closed without merging**